.github/workflows/nodejs.yml

```
@@ -1,28 +1,104 @@
+        path: |
+          dist/
+          node_modules/.cache/
+        key: ${{ runner.os }}-build-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('src/**/*') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ hashFiles('**/package-lock.json') }}-
+          ${{ runner.os }}-build-
 
+    - name: Format code
+      run: npm run fmt
+
+    - name: Lint code  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)